iOS/PixelDefinitions/pixels/data_broker_protection.json5

```diff
@@ -390,6 +390,12 @@
                         "type": "integer"
                     }
                 }
+            },
+            {
+                "key": "isAuthenticated",
+                "description": "Whether the user is authenticated",
+                "type": "string",
+                "enum": ["true", "false"]
```

Comment:
Learned a trick from my current PR privacy review: `type` can be `boolean` instead of an `enum`. Any reason why we should go for `enum` here instead? I’m also asking for my own PR 😁